### PR TITLE
Added power support

### PR DESCRIPTION
**Pull request by dthadi3**
Added power support for the travis.yml file with ppc64le. This is part of the Ubuntu distribution for ppc64le. This helps us simplify testing later when distributions are re-building and re-releasing. For more info tag @gerrith3.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 sudo: false
+arch:
+  - AMD64
+  - ppc64le
 go:
   - 1.4
   - 1.5
   - 1.6
   - tip
+# Disable version 1.4
```
Comment (lucab) [inline .travis.yml:11]: For context, why is is 1.4 specifically singled out here?

Overall, all of these are ancient version anyway, so I'd be ok with dropping the oldest ones and just keep 1.6.

Comment (dthadi3) [reply .travis.yml:11]: yeah ok, please go ahead dropping the oldest ones and keep 1.6. Thanks

Comment (lucab) [reply .travis.yml:11]: Feel free to just amend this PR, dropping 1.4 & 1.5 from the list above instead of adding this `exclude` entry.

Comment (dthadi3) [reply .travis.yml]: Dropped 1.4 & 1.5 Go versions and deleted exclude jobs entry too....

Comment (dthadi3) [reply .travis.yml]: If agreed with the current changes, please accept the build & resolve the conversation status

**Comment by lucab** [comment]
LGTM, just squash everything into a single commit and I'll merge. Thanks again for this PR!

**Comment by dthadi3** [comment]
Thanks for your suggestion, Due to some technical issue i'm closing this PR. I raised other PR #31 please accept that PR and Merge it.

**Pull request closed without merging**